_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -31,6 +31,8 @@ parent: Sercomm
 | Serial encoding | 8-N-1                  |
 | Form Factor     | ONT                    |
 
+## Credits
+This whole documentation here was made possible by reverse engineering, and time investment from [hwti](https://github.com/hwti) and the rest of the folks from the forum mention in the links section of the page. Thanks a lot !
```

Comment:
This paragraph catches me a little off guard, it's the first time it's happened to us, and generally this has always been more sufficent: @nanomad @ErnyTech 

![image](https://github.com/hack-gpon/hack-gpon.github.io/assets/26844016/3c9a478e-33a7-48e1-9984-fbbaa4c93203)

it is also true that if @hwti does not do a PR it will never appear.

We have to think about how and where to put any credits, and whether the way you propose is OK.

Let's be clear, it is not that we are against credits, only that it must be a considered thing and consistent with the rest of the wiki.

Reply:
I totally get this, yet I would felt I'm steeling the work of other not being able to thanks/give credit to whom made it possible.

Reply:
With @ErnyTech and @nanomad we're reasoning on a path that pleases everybody, certainly not so at the top of the page, and no it's not belittling, it's just that it's a wiki not a post on a forum

Reply:
In any case, mish links are really for citing sources without a paper-like formalism

Reply:
I don't know exactly how copyright would apply to information like this.

All content here comes from a shared work from mostly @benoitm974 and me, and a few other people on lafibre.info.
The goal was to help anyone wanting to use this ONT, and the wiki is the best place to sum up all our discoveries and make them available for more users (for example in Germany).

Reply:
What we want to know is whether besides @benoitm974 anyone else wants to be on the list of authors. And/or if we should put all of lafibre.info

Copyright side we want to ensure that we don't put for example a copyrighted guide but only copy the steps and remove superfluous parts

Reply:
The commands come from a shared reverse-engineering work :
 - analysis of web interface requests
 - NAND dump, ECC correction, rootfs extraction
 - search for vulnerabilities in the webserver binaries, allowing to run commands
 - search for commands to change the GPON/OMCI settings
 - various analysis on how the ONT boots (not yet explained here, but it only matters if we are able to flash a modified firmware, which is uncertain and currently not really needed) and works

But after a command is found there is no copyright on it, the only part where different people could have come to different solutions is the vulnerability.

So, since the text from this PR is from @benoitm974, I think that from a copyright point of view he is the author of the guide.

Then, it's up to you on whether to acknowledge the reverse-engineering work, and if so how.

Reply:
For me you can open a PR dummy by inserting a new line.